testdata/project-v3-addon/README.md

```diff
@@ -0,0 +1,77 @@
+# project-v3-addon
+// TODO(user): Add simple overview of use/purpose
+
+## Description
+// TODO(user): An in-depth paragraph about your project and overview of use
+
+## Getting Started
+You’ll need a Kubernetes cluster to run against. You can use [KIND](https://sigs.k8s.io/kind) to get a local cluster for testing, or run against a remote cluster.
+**Note:** Your controller will automatically use the current context in your kubeconfig file (i.e. whatever cluster `kubectl cluster-info` shows).
+
+### Running on the cluster
+1. Install Instances of Custom Resources:
+	```sh
+	kubectl apply -f config/samples/
+	```	
+2. Build and push your image to the location specified by `IMG`:
+	```sh
+	make docker-build docker-push IMG=<some-registry>/<project-name>:tag
+	```	
+3. Deploy the controller to the cluster with the image specified by `IMG`:
+	```sh
+	make deploy IMG=<some-registry>/project-name>:tag
+	```	 
+### Uninstall CRDs
+To delete the CRDs from the cluster:
+	```sh
+	make uninstall
+	```	
```

Comment:
Do we need the space?
```suggestion

```sh
make uninstall
```	
```

Could not it be only ^?

Reply:
Should be fixed in https://github.com/kubernetes-sigs/kubebuilder/pull/2610/commits/a0faa6f08895abaf1bad432af586e49ef8ce107e